beacon-chain/archiver/service.go

```diff
@@ -101,60 +100,18 @@ func (s *Service) archiveCommitteeInfo(ctx context.Context, headState *pb.Beacon
 
 // We archive active validator set changes that happened during the epoch.
 func (s *Service) archiveActiveSetChanges(ctx context.Context, headState *pb.BeaconState) error {
```

Comment:
how often are you going to be calling this function ? 
this loops through the validator set 3 times now , which 
might be a problem if this is called frequently

Reply:
Only once per epoch, and it's O(3N) = O(N), so it should be alright